include/ofxCore.h

```diff
@@ -901,7 +917,7 @@ General status codes start at 1 and continue until 999
 /** @brief Status code indicating all was fine */
 #define kOfxStatOK 0
 
-/** @brief Status error code for a failed operation */
+/** @brief Status error code for a failed operation. During kOfxActionLoad if possible can you assume returning kOfxStatFailed is done silently and is to be interpreted as plug-in is explicitetly not supporting this Host.Version as opposed to an actual error */
```

Comment:
I don't think this comment is necesssary here -- kOfxStatFailed is very general and we shouldn't discuss its meaning in any particular context here.